footer-featured.php

```diff
@@ -1,7 +1,7 @@
 			<div id="footer">
 				<div class="row">
 					<div class="span12 text-center">
-						<?=wp_nav_menu( array(
+						<?php echo wp_nav_menu( array (
```

Comment:
`echo` isn't necessary when calling `wp_nav_menu()`; this function echo's its results by default